### PR TITLE
feature/addNote

### PR DESCRIPTION
**Pull request by Alphawave1989**

### CODE REVIEW

File: components/PlantDetail.js [hunk 1]
```diff
@@ -67,6 +69,15 @@ export default function PlantDetail({
           handleCancel={handleCancel}
         />
       )}
+      <NotesContainer>
+        <FormComments
+          onSubmit={handleAddNotes}
```
Comment (MerleMak) [inline components/PlantDetail.js:74]:
```suggestion
          onSubmit={onAddNotes}
```
It it a common practice to use the Prefix "on" up until the point where the function is actually defined. :)

File: pages/plants/[id]/index.js [hunk 2]
```diff
@@ -47,6 +48,7 @@ export default function DetailPage({
         handleToggleOwnedPlants={handleToggleOwnedPlants}
         handleDeletePlant={handleDeletePlant}
         handleEditPlant={handleEditPlant}
+        handleAddNotes={handleAddNotes}
```
Comment (MerleMak) [inline pages/plants/[id]/index.js:51]:
```suggestion
       onAddNotes={onAddNotes}
```

File: pages/_app.js [hunk 3]
```diff
@@ -126,6 +145,7 @@ export default function App({ Component, pageProps }) {
             handleDeletePlant={handleDeletePlant}
             handleEditPlant={handleEditPlant}
             landingData={landingData}
+            handleAddNotes={handleAddNotes}
```
Comment (MerleMak) [inline pages/_app.js:148]:
```suggestion
            onAddNotes={handleAddNotes}
```

**Review by MerleMak — COMMENTED** [review]
Great work!
I left you some comments you might want to consider :)
Keep it up! Really nice :)

File: pages/_app.js [hunk 4]
```diff
@@ -104,6 +104,25 @@ export default function App({ Component, pageProps }) {
     }
   }
 
+  async function handleAddNotes(plant, id, mutate) {
+    console.log("Added note");
+
+    const respone = await fetch(`/api/plants/${id}`, {
+      method: "PUT",
```
[the remark this answers is not in the document]
Comment (Alphawave1989) [reply pages/_app.js:111]: We update our plant, that's why we need "PUT" instead of "POST".

[the remark this answers is not in the document]
Comment (MerleMak) [reply pages/_app.js:109]: oh yeah sorry, i misunderstood!

**Review by MerleMak — APPROVED** [review]
perfect :)

**Pull request merged**